Makefile

```
@@ -157,12 +157,19 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+
+.PHONY: markdownlint
+# the markdownlint  target requires
+# markdownlint-cli2 to be available locally in the PATH
+markdownlint: ## Check formatting of documentation sources
+	@markdownlint-cli2 "Documentation/**.md" "#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
```

```suggestion
	@markdownlint-cli2 "Documentation/**.md" "#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
```

not sure just a suggestion can we add a log message that this requires the markdownlint-cli2 to be installed on local system PATH or like check if the tool is available if not log error or else continue?

something like that? but otherwise looks good to me

---

done.